include/bluetooth/services/ots.h

```diff
@@ -468,6 +470,76 @@ struct bt_ots_feat {
 	uint32_t olcp;
 } __packed;
 
+/** @brief Object metadata request bit field values */
+enum {
+	/** @brief Request object name */
+	BT_OTS_METADATA_REQ_NAME                        = BIT(0),
+	/** @brief Request object type */
+	BT_OTS_METADATA_REQ_TYPE                        = BIT(1),
+	/** @brief Request object size */
+	BT_OTS_METADATA_REQ_SIZE                        = BIT(2),
+	/** @brief Request object first created time */
+	BT_OTS_METADATA_REQ_CREATED                     = BIT(3),
+	/** @brief Request object last modified time */
+	BT_OTS_METADATA_REQ_MODIFIED                    = BIT(4),
+	/** @brief Request object ID */
+	BT_OTS_METADATA_REQ_ID                          = BIT(5),
+	/** @brief Request object properties */
+	BT_OTS_METADATA_REQ_PROPS                       = BIT(6),
+	/** @brief Request all object metadata */
+	BT_OTS_METADATA_REQ_ALL                         = 0x7F,
+};
+
+/** @brief Date and Time structure */
+struct bt_ots_date_time {
+	uint16_t year;
+	uint8_t month;
+	uint8_t day;
+	uint8_t hours;
+	uint8_t minutes;
+	uint8_t seconds;
+};
+#define DATE_TIME_FIELD_SIZE 7
```

Comment:
Could this be moved to an internal header? 

```suggestion
#define BT_OTS_DATE_TIME_FIELD_SIZE 7
```